indico/modules/events/timetable/controllers/display.py

```diff
@@ -47,8 +48,9 @@ def _process(self):
         if self.theme is None:
             event_info = serialize_event_info(self.event_new)
             timetable_data = TimetableSerializer().serialize_timetable(self.event_new, strip_empty_days=True)
+            timetable_settings = layout_settings.get(self.event_new, 'timetable_theme_settings', {})
```

Comment:
do we ever set them to None? if not the `, {}` is not needed since it's the default value anyway which is used if there's no value for the setting